manifests/k/KINGSGROUP/KOA/1.0/KINGSGROUP.KOA.yaml

```
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.4.0.schema.json
+
+PackageIdentifier: KINGSGROUP.KOA
+PackageVersion: 1.0

```

```suggestion
PackageVersion: 1.0.6.38
```
This should match Control Panel